src/modules/reading-room/components/AddToCollectionBlade/AddToCollectionBlade.tsx

```diff
@@ -281,7 +284,7 @@ const AddToCollectionBlade: FC<AddToCollectionBladeProps> = (props) => {
 
 								<li className={styles['c-add-to-collection-blade__list-button']}>
 									<CreateCollectionButton
-										afterSubmit={() => collections.refetch()}
+										afterSubmit={() => getCollections.refetch()}
```

Comment:
can be simplified, or does typescript compain when you do this?
```suggestion
										afterSubmit={getCollections.refetch}
```